interface/language/language.php

```diff
@@ -61,6 +61,6 @@
 endswitch;
 ?>
 
-<BR><A HREF="lang.info.html" class='cp-misc' TARGET="_blank"><?php echo htmlspecialchars(xl('Info'),ENT_NOQUOTES); ?></A>
+<BR><A HREF="lang.info.html" TARGET="_blank"><?php echo htmlspecialchars(xl('Info'),ENT_NOQUOTES); ?></A>
```

Comment:
Stripping off these is a baaaaadddd idea. I know but this is not what we intended for when placing this cause we didn't finish this prior to gsoc. I guess team alpha will be coming on board any time soon to re-tackle this. IMO let them be cause we aim at making stuff uniform once gci is over

Reply:
Also there will be a removal of themes from the codebase and we will just be left with 2 themes.

Reply:
@muarachmann this is in style.php so it applies to all themes. Is there really nothing that can be done for this specifically?

Reply:
What i am saying is we will do a batch revamp of the buttons across the whole EHR at once. You are welcome to join if interested.And for that we will need those `cp-xxxx` classes. We started but got shut cause of  gsoc

Reply:
@muarachmann I see. closing this then. I will definitely see if I have time to contribute.

Reply:
Thanks @jonathanzhang53 we will appreciate your help. You are doing great so far i like your debug skills though 👍